spark/src/main/java/org/opensearch/sql/spark/transport/TransportCancelAsyncQueryRequestAction.java

```diff
@@ -40,6 +42,7 @@ protected void doExecute(
       Task task,
       CancelAsyncQueryActionRequest request,
       ActionListener<CancelAsyncQueryActionResponse> listener) {
+    MetricUtils.incrementNumericalMetric(MetricName.ASYNC_QUERY_CANCEL_API_REQUEST_COUNT);
```

Comment:
we should add REQ_COUNT in rest layer?
* SUCCESS_CNT
* CUSTOMER_FAIL_CNT
* SYS_FAIL_CNT

Reply:
Addressed.